timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -302,6 +304,18 @@ def _get_estimator(self) -> GluonTSEstimator:
             **init_args,
         )
 
+    def _is_gpu_available(self) -> bool:
+        import torch.cuda
+
+        return torch.cuda.is_available()
+
+    def get_minimum_resources(self, is_gpu_available: bool = False) -> Dict[str, Union[int, float]]:
```

Comment:
Is my understanding correct?
- `get_minimum_resources` method is necessary to tell Ray "please always assign 1 GPU to this job if there is GPU available"
- all GluonTS models will always try to use the GPU if GPU is available, regardless of `num_gpus` passed to `model.fit` or anything else

Reply:
That matches my understanding, yes.

Reply:
Thanks!